docs/en/siem/machine-learning.asciidoc

```diff
@@ -21,35 +21,185 @@ image::ml-ui.png[]
 === Manage machine learning jobs
 For users with the `ml_admin` role, the `Anomaly Detection` interface within
 the main navigation header can be used for for viewing, starting, and stopping
-SIEM Machine Learning Jobs.
+SIEM machine learning jobs.
+
+TIP: To add a custom job to the `Anomaly Detection` interface, add a `SIEM` tag to
+the job's `Group` field ({kib} -> {ml-cap} -> Create/Edit job -> Job details).
 
 [float]
 [[included-jobs]]
 ==== Prebuilt Jobs
 
 The SIEM app ships with prebuilt Machine Learning Jobs for detecting anomalies.
-If your environment is configured with the appropriate indices (`auditbeat-*`
-and `winlogbeat-*`) via {kib} -> Management -> Index Patterns, the jobs will be
-installed on page load, and will be displayed within the `Anomaly Detection`
-interface.
+If you ship data using https://www.elastic.co/products/beats[Beats] and your environment
+is configured with the appropriate indices (`auditbeat-*`, `filebeat-*`, `packetbeat-*`,
+or `winlogbeat-*`) via {kib} -> Management -> Index Patterns, the jobs will be installed
+on page load, and will be displayed within the `Anomaly Detection` interface.
+
+NOTE: Machine learning jobs look back and analyse two weeks of historical data prior to
+the time they are enabled. After jobs are enabled, they continuously analyse incoming data.
```

Comment:
two weeks is the conservative default which can be increased and should be increased when capacity permits - the more data the models longest, they better they get.

Reply:
@randomuserid 
How is this changed?

Reply:
Merging as is, we'll update this info afterwards.